cmake/modules/FindLibCXX.cmake

```diff
@@ -0,0 +1,15 @@
+# Find libc++
+
```

Comment:
Please add the standard comment block with the variables setup and their values.